### PR TITLE
Add constants from pokecrystal

### LINKED ISSUE #26: More human-friendly names for some constants
Taking inspiration from [pokecrystal's constants](https://github.com/pret/pokecrystal/blob/2fe0cbbb19df504723934f39473064033c64ef6f/constants/gfx_constants.asm), for example. This would allow them to use `hardware.inc`, as well. (cc @Rangi42)

### PR DESCRIPTION
**Pull request by rawr51919**
This PR is a proof-of-concept showing what a `hardware.inc` using constants from pokecrystal would look like.
Some of these names can be used for new names for constants down the line, so this one might be a little while before any action comes of this.
Fixes #26

### CODE REVIEW

**Comment by eievui5** [comment]
I don't know if there was prior discussion about this, but I think this would really clutter the file simply to become compatible with pokecrystal. Why not update pokecrystal to conform to hardware.inc instead?

**Comment by coltongit** [comment]
> I don't know if there was prior discussion about this, but I think this would really clutter the file simply to become compatible with pokecrystal. Why not update pokecrystal to conform to hardware.inc instead?

There wasn't much in the way of discussion about this, yet pokecrystal's PR #943 does exactly that

**Comment by eievui5** [comment]
I think rather than updating this file to support every different project's naming scheme, we should get the projects to settle on using hardware.inc

**Comment by coltongit** [comment]
Backported a constant addition from this PR to #33, `RPB_LED_ON`. I had forgot to add it during #31

**Comment by coltongit** [comment]
> I think rather than updating this file to support every different project's naming scheme, we should get the projects to settle on using hardware.inc

There has also been an issue opened on the subject on pokecrystal issue [#914](https://github.com/pret/pokecrystal/issues/914)

File: hardware.inc [hunk 1]
```diff
@@ -827,17 +861,26 @@ DEF rPCM34 EQU $FF77
...
+DEF IEB_TIMER   EQU 2
+DEF IEB_STAT    EQU 1
+DEF IEB_VBLANK  EQU 0
+DEF IEB_DEFAULT EQU (1 << IEB_SERIAL) | (1 << IEB_TIMER) | (1 << IEB_STAT) | (1 << IEB_VBLANK)
+
+DEF JOYPAD      EQU IEB_HILO
+DEF SERIAL      EQU IEB_SERIAL
+DEF TIMER       EQU IEB_TIMER
+DEF LCD_STAT    EQU IEB_STAT
+DEF VBLANK      EQU IEB_VBLANK
```
[the remark this answers is not in the document]
Comment (avivace) [reply hardware.inc:883]: Agree here

**Pull request closed without merging**